### PR TITLE
Add Zowe Mobile & Zowe Client SDK as incubation

### PR DESCRIPTION
**Pull request by solsu01**
Separated sub-projects to Active and Incubation.
Added Zowe Mobile & Zowe Client SDK as incubation.
Added Zowe Mobile incubation responsibilities to Zowe Application Framework squad.
Added Zowe Client SDK incubation responsibilities to Zowe CLI squad.
Added slack channels for both.

### CODE REVIEW

**Comment by PeterFandelAtRocket** [comment]
Looks good to me Sujay - thanks

File: README.md [hunk 1]
```diff
@@ -3,10 +3,14 @@
...
-    - [Zowe Application Framework](#zowe-application-framework)
-    - [Zowe Explorer](#zowe-explorer)
+    - [Active](#active)
+       - [Zowe API Mediation Layer](#zowe-api-mediation-layer)
+       - [Zowe CLI](#zowe-cli)
+       - [Zowe Application Framework](#zowe-application-framework)
+       - [Zowe Explorer](#zowe-explorer)
+    - [Incubation](#incubation)
+       - [Zowe Mobile](#zowe-mobile)
+       - [Zowe Client SDK](zowe-client-sdk)
```
[the remark this answers is not in the document]
Comment (solsu01) [reply README.md]: Thanks @jellypuno - I noticed that i was missing the # to make it a link. This is the link to the sub-section further down in the document. 

The slack channel is listed further down in the Slack section.

**Comment by zFernand0** [comment]
LGTM!
I noticed the Zowe Explorer Squad link (from the ToC) isn't working properly :cry:
Also, we should give the [DCO GitHub UI](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo?hl=en) chrome extension a try in order to avoid manually approving DCOs 😋

**Comment by solsu01** [comment]
> LGTM!
> I noticed the Zowe Explorer Squad link (from the ToC) isn't working properly 😢
> Also, we should give the [DCO GitHub UI](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo?hl=en) chrome extension a try in order to avoid manually approving DCOs 😋

Fixed the squad link. 

I am using the chrome extension now and the latest commits have proper DCO. Before that I was trying to do it through git locally and for some reason, this repo does not clone properly. I haven't been able to rebase/fix earlier commits due to this. Manual DCO is needed if even 1 commit doesn't have proper sign-off.

**Pull request merged**